### PR TITLE
fix links and pictures

### PR DESCRIPTION
**Pull request by syjabri**

### CODE REVIEW

File: docs/_imke/clusterlifecycle/creatingacluster/index.de.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ auswählen:
 Man kann ein existierendes Netzwerk verwenden, um den Cluster zu erstellen. Dazu müssen wir das Netzwerk und das Subnetz auswählen.
 Diese müssen mit einem Router verbunden sein.
 Also man soll einen Router erstellen, was auf dem Optimist Dashboard oder aus der OpenStack Kommandozeile durchgeführt werden kann.
-Hierfür kann man die OpenStack Dokumentation nutzen, um den Router zu erstellen und das Netzwerk zu verbinden. (<http://docs.innovo.cloud/guided_tour/en/step10/>)
+Hierfür kann man die OpenStack Dokumentation nutzen, um den Router zu erstellen und das Netzwerk zu verbinden. (<https://docs.innovo.cloud/guided_tour/de/schritt10/>)
```
Comment (qeqar) [inline docs/_imke/clusterlifecycle/creatingacluster/index.de.md:37]: Shouldn't we use the new docu link here?

File: docs/_imke/clusterlifecycle/upgradingacluster/index.de.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Infrastruktur sicher zu halten. In diesem Fall kann der nächste
 Block übersprungen werden. Nodes müssen dann nämlich dennoch
 selbst geupdated werden.
 
-Bevor Sie einen Cluster upgraden, lesen Sie bitte das [Changelog](/imke/about/kubernetesversions) der Ziel-Kubernetes-Version,
+Bevor Sie einen Cluster upgraden, lesen Sie bitte das [Changelog](/imke/about/changelog-v2.14.8/) der Ziel-Kubernetes-Version,
```
Comment (qeqar) [inline docs/_imke/clusterlifecycle/upgradingacluster/index.de.md:20]: Why use a distinct version here -> we need to update it on any new changelog?

File: docs/_imke/clusterlifecycle/upgradingacluster/index.en.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ own infrastructure up-to-date. In this case the following section
 *The Cluster* can be skipped. However, nodes must still be updated
 manually by you.
 
-Before you upgrade a cluster, please refer to the target version's [Changelog](/imke/about/kubernetesversions)
+Before you upgrade a cluster, please refer to the target version's [Changelog](/imke/about/changelog-v2.14.8)
```
Comment (qeqar) [inline docs/_imke/clusterlifecycle/upgradingacluster/index.en.md:19]: Same as DE version

**Review by qeqar — CHANGES_REQUESTED** [review]

File: docs/_imke/clusterlifecycle/creatingacluster/index.de.md (hunk 1 above)
Comment (syjabri) [reply docs/_imke/clusterlifecycle/creatingacluster/index.de.md:37]: i couldn't find the new one in the optimist page

File: docs/_imke/clusterlifecycle/upgradingacluster/index.de.md (hunk 2 above)
Comment (syjabri) [reply docs/_imke/clusterlifecycle/upgradingacluster/index.de.md:20]: we don't have a global page

File: docs/_imke/clusterlifecycle/creatingacluster/index.de.md (hunk 1 above)
Comment (qeqar) [reply docs/_imke/clusterlifecycle/creatingacluster/index.de.md:37]: Ahh not jet merged! it is in #5
Maybe we can already update to it so we don't forget it?

File: docs/_imke/clusterlifecycle/upgradingacluster/index.de.md (hunk 2 above)
Comment (qeqar) [reply docs/_imke/clusterlifecycle/upgradingacluster/index.de.md:20]: then we should probably link to about/ if this is possible

**Review by qeqar — APPROVED** [review]
/LGTM

**Pull request merged**